src/main/scala/org/ergoplatform/api/ScriptApiRoute.scala

```diff
@@ -91,6 +114,74 @@ case class ScriptApiRoute(readersHolder: ActorRef, ergoSettings: ErgoSettings)
     }
   }
 
+
+  case class ExecuteRequest(script: String,
+                            env: Map[String,Any],
+                            ctx: ErgoLikeContext)
+    extends TransactionRequest
+
+
+  implicit final val decodeMap: Decoder[Any] = new Decoder[Any] {
+    final def apply(c: HCursor): Result[Any] = {
+      val x = DataJsonEncoder.decode(c.value)
+      Right(x)
+    }
+  }
+
+  class ExecuteRequestDecoder(settings: ErgoSettings) extends Decoder[ExecuteRequest] with JsonCodecs {
+    def apply(cursor: HCursor): Decoder.Result[ExecuteRequest] = {
+      for {
+        script <- cursor.downField("script").as[String]
+        env <- cursor.downField("env").as[Map[String,Any]]
```

Comment:
@greenhat I think here should be Map[String, AnyValue], so AnyValueDecoder should work